code/datums/diseases/advance/symptoms/genetics.dm

```diff
@@ -43,36 +56,15 @@ Bonus
 	switch(A.stage)
 		if(4, 5)
 			to_chat(C, "<span class='warning'>[pick("Your skin feels itchy.", "You feel light headed.")]</span>")
-			C.dna.remove_mutation_group(possible_mutations)
-			for(var/i in 1 to power)
-				C.randmut(possible_mutations)
-
-// Archive their DNA before they were infected.
-/datum/symptom/genetic_mutation/Start(datum/disease/advance/A)
-	if(!..())
-		return
-	if(A.properties["stealth"] >= 5) //don't restore dna after curing
-		no_reset = TRUE
-	if(A.properties["stage_rate"] >= 10) //mutate more often
-		symptom_delay_min = 20
-		symptom_delay_max = 60
-	if(A.properties["resistance"] >= 8) //mutate twice
-		power = 2
-	possible_mutations = (GLOB.bad_mutations | GLOB.not_good_mutations) - GLOB.all_mutations[RACEMUT]
-	var/mob/living/carbon/M = A.affected_mob
-	if(M)
-		if(!M.has_dna())
-			return
-		archived_dna = M.dna.mutation_index
+			if(badtothebone)

```

Comment:
you can do something like this

`C.easy_randmut(NEGATIVE + MINOR_NEGATIVE + POSITIVE - nopositives , TRUE, TRUE, TRUE, mutadone_proof)`

Since a TRUE as nopositives is equivalent to the value of the POSITIVE define ;)

Reply:
you could also reformat the var so it describes what traits to remove then set it as

`muts_to_not_supply = POSITIVE` if making the assumption POSITIVE and TRUE will always be 1 is bad.